sys-devel/llvm/llvm-18.0.0_pre20230906.ebuild

```diff
@@ -16,13 +16,34 @@ HOMEPAGE="https://llvm.org/"
 # 2. xxhash: BSD.
 # 3. MD5 code: public-domain.
 # 4. ConvertUTF.h: TODO.
+# 5. MSVCSetupApi.h: MIT
+# 6. sorttable.js: MIT
 
-LICENSE="Apache-2.0-with-LLVM-exceptions UoI-NCSA BSD public-domain rc"
+LICENSE="Apache-2.0-with-LLVM-exceptions UoI-NCSA BSD MIT public-domain rc"
 SLOT="${LLVM_MAJOR}/${LLVM_SOABI}"
 KEYWORDS=""
 IUSE="
-	+binutils-plugin +debug debuginfod doc exegesis libedit +libffi
-	ncurses test xar xml z3 zstd
+	+binutils-plugin +debug debuginfod doc exegesis ieee-long-double libedit
+	+libffi ncurses +pie +static-analyzer test xar xml z3 zstd
+"
```

Comment:
Missing `extras`: (ditto for .9999)

```
 * Messages for package sys-devel/llvm-18.0.0_pre20230906:

 * FAILED postinst: 1
 * You can find additional opt-viewer utility scripts in:
 *   /usr/lib/llvm/18/share/opt-viewer
 * To use these scripts, you will need Python along with the following
 * packages:
 *   dev-python/pygments (for opt-viewer)
 *   dev-python/pyyaml (for all of them)
 * You can find additional utility scripts in:
 *   /usr/lib/llvm/18/share/clang
 * ERROR: sys-devel/llvm-18.0.0_pre20230906::llvm-monorepo failed (postinst phase):
 * 
 * Call stack:
 *          ebuild.sh, line  136:  Called pkg_postinst
 *        environment, line 3711:  Called use 'extra'
 *   phase-helpers.sh, line  256:  Called die
 * The specific snippet of code:
 *   				die "USE Flag '${u}' not in IUSE for ${CATEGORY}/${PF}"
 * 
 * If you need support, post the output of `emerge --info '=sys-devel/llvm-18.0.0_pre20230906::llvm-monorepo'`,
 * the complete build log and the output of `emerge -pqv '=sys-devel/llvm-18.0.0_pre20230906::llvm-monorepo'`.
 * The complete build log is located at '/mnt/scratch/portage/sys-devel/llvm-18.0.0_pre20230906/temp/build.log'.
 * The ebuild environment file is located at '/mnt/scratch/portage/sys-devel/llvm-18.0.0_pre20230906/temp/environment'.
 * Working directory: '/mnt/scratch/portage/sys-devel/llvm-18.0.0_pre20230906/empty'
 * S: '/mnt/scratch/portage/sys-devel/llvm-18.0.0_pre20230906/work/llvm'
```

Reply:
> Missing `extras`: (ditto for .9999)
> 
> ```
>  * Messages for package sys-devel/llvm-18.0.0_pre20230906:
> 
>  * FAILED postinst: 1
>  * You can find additional opt-viewer utility scripts in:
>  *   /usr/lib/llvm/18/share/opt-viewer
>  * To use these scripts, you will need Python along with the following
>  * packages:
>  *   dev-python/pygments (for opt-viewer)
>  *   dev-python/pyyaml (for all of them)
>  * You can find additional utility scripts in:
>  *   /usr/lib/llvm/18/share/clang
>  * ERROR: sys-devel/llvm-18.0.0_pre20230906::llvm-monorepo failed (postinst phase):
>  * 
>  * Call stack:
>  *          ebuild.sh, line  136:  Called pkg_postinst
>  *        environment, line 3711:  Called use 'extra'
>  *   phase-helpers.sh, line  256:  Called die
>  * The specific snippet of code:
>  *   				die "USE Flag '${u}' not in IUSE for ${CATEGORY}/${PF}"
>  * 
>  * If you need support, post the output of `emerge --info '=sys-devel/llvm-18.0.0_pre20230906::llvm-monorepo'`,
>  * the complete build log and the output of `emerge -pqv '=sys-devel/llvm-18.0.0_pre20230906::llvm-monorepo'`.
>  * The complete build log is located at '/mnt/scratch/portage/sys-devel/llvm-18.0.0_pre20230906/temp/build.log'.
>  * The ebuild environment file is located at '/mnt/scratch/portage/sys-devel/llvm-18.0.0_pre20230906/temp/environment'.
>  * Working directory: '/mnt/scratch/portage/sys-devel/llvm-18.0.0_pre20230906/empty'
>  * S: '/mnt/scratch/portage/sys-devel/llvm-18.0.0_pre20230906/work/llvm'
> ```

ah, this is due to me copy-pasting pkg_postinst from clang without changing the use. I'll fix asap.